dev-tool.sh

```
@@ -44,39 +51,98 @@ choose() {
+  mvn -pl "$ACTIVE_MODULE" exec:java -Dexec.args="${quoted% }"
 }
 
 # Autocompletion function for 'choose'
 _choose_completion() {
   local cur opts
   COMPREPLY=()
   cur="${COMP_WORDS[COMP_CWORD]}"
-  opts=$(list)
+  opts=$(_modules_raw)
 
   if [ ${COMP_CWORD} -eq 1 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)